nixos/modules/services/networking/unbound.nix

```diff
@@ -95,6 +105,16 @@ in {
         '';
       };
 
+      tags = mkOption {
```

Comment:
feels like this would be better placed in the settings section as `server.define-tag` to avoid users defining it twice by accident. though that would require even more special-case code to not generate it twice by accident, which is somewhat less than ideal

Reply:
Is that better?

Reply:
yes :+1: